test/versioned/iovalkey/iovalkey.test.js

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const test = require('node:test')
+const assert = require('node:assert')
+const helper = require('../../lib/agent_helper')
+const params = require('../../lib/params')
+const urltils = require('../../../lib/util/urltils')
+const { tspl } = require('@matteo.collina/tspl')
+const { assertMetrics, assertPackageMetrics } = require('../../lib/custom-assertions')
+const { removeModules } = require('../../lib/cache-buster')
+
+// Indicates unique database in Valkey. 0-15 supported.
+const DB_INDEX = 3
```

Comment:
our team can clean these up but this index is being used elsewhere.  can you change this value to `4`